lib/Dialect/Secret/Conversions/SecretToBGV/SecretToBGV.cpp

```diff
@@ -178,7 +191,13 @@ struct SecretToBGV : public impl::SecretToBGVBase<SecretToBGV> {
       module->emitError(
           "expected batched secret types to be tensors with dimension "
           "matching ring parameter");
-      return signalPassFailure();
+      signalPassFailure();
```

Comment:
Why the change in style here? Does the linter complain about the `return voidFunction()`?

Reply:
The linter doesn't complain, but no point in returning a void. Just adds confusion since you would assume it returns something